types/to-regex/to-regex-tests.ts

```diff
@@ -0,0 +1,16 @@
+import toRegex from 'to-regex';
+
+toRegex('foo');
+
+toRegex(['foo', 'bar']);
+
+toRegex('foo', { contains: true });
+
+toRegex('foo', {
+    contains: true,
+    negate: false,
+    nocase: true,
+    flags: 'g',
+    cache: true,
+    safe: true
+});
```

Comment:
```suggestion
});

```

Add a newline at the end of the file